src/plugin.js

```diff
@@ -138,6 +138,18 @@ class PlaylistMenuItem extends Component {
     const li = document.createElement('li');
     const item = this.options_.item;
 
+    if (item.data) {
+      const dataEntries = Object.entries(item.data);
```

Comment:
Unfortunately `Object.entries`, is not supported in IE and we still support IE 11. I think that you will have to use `Object.keys` and then manually get the value with `item.data[key]` below.

Reply:
@BrandonOCasey - You just saved me a cross-browser issue. Hah. Thanks! Update to this PR inbound shortly, btw.